images/kcp-registrar/data/syncer-patch.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Minor point. But let's maintain consistency with the yaml extension. Plus [official recommendation](https://yaml.org/faq.html) as well :)
.yml -> .yaml